Appearance tweaks and studio tests, per QA Feedback.

Kelketek opened the pull request.

Kelketek committed:
Appearance tweaks, per QA Feedback.

Kelketek commented on poll/public/css/poll.css:
```diff
@@ -145,7 +145,7 @@ li.poll-result .poll-image {
 .survey-table thead tr th {
     font-weight: bold;
-    font-size: .8rem;
+    font-size: .9rem;
```
The QA folks requested that the answer text be bold. It was already bold, so I'm guessing that they just wanted it a bit larger. Please let me know if this is in error.

antoviaque commented on poll/public/js/poll_edit.js:
```diff
@@ -26,6 +26,10 @@ function PollEditUtil(runtime, element, pollType) {
+                            // We have to insert this key now rather than keep it in the template
+                            // so that it's generated with the current time. If we constructed it as part
+                            // of the template, all the keys would be the same, since the time would be calculated
+                            // once.
```
Ah, I see that you already added the comment here - great.

My comment about using unique ids instead of dates still stands though.

antoviaque commented:
@Kelketek For the tests - to test the studio view, it's mostly a matter of being able to load the studio view in the workbench, right? Have you looked at https://github.com/edx/xblock-sdk/blob/4acdbe311d81feefce61c87671cdffb070f26f53/workbench/views.py#L48 and https://github.com/edx/xblock-sdk/blob/3957e4fec68cb0d55e9ed97668ab920f9704ac43/workbench/urls.py#L16-L20 ?

Kelketek commented:
@xavierantoviaque No, I wasn't aware of that. This makes me happy, as I wasn't even aware I could make tests for studio. Though it also means I have a few more tests to write :)

Kelketek committed:
Added studio tests.

Kelketek commented:
@antoviaque Studio tests added for the requested scenario.

antoviaque commented:
@Kelketek Thanks for adding the tests!

What remains now on my side is my comment about id generation: https://github.com/mckinseyacademy/xblock-poll/pull/2/files#r23437679 https://github.com/mckinseyacademy/xblock-poll/pull/1#discussion_r23437520

mtyaka commented on poll/public/js/poll_edit.js:
```diff
@@ -1,12 +1,16 @@
     this.loadQuestions = runtime.handlerUrl(element, 'load_questions');
 
+    // The workbench doesn't support notifications.
+    notify = typeof(runtime.notify) != 'undefined';
```
I think you also need to add an `if (notify)`check just above line 20: https://github.com/open-craft/xblock-poll/blob/style_tweaks/poll/public/js/poll_edit.js#L20

mtyaka commented:
I think an `if (notify)` check is missing, but otherwise looks good to me :+1:

Kelketek commented:
@mtyaka I'm not sure what good it would do there, since the SDK has no concept of cancellation. I can add it, but it will make the button a no-op, and it seems more appropriate to me that an exception should be raised, as it is now, since if someone is expecting the SDK to cancel, they have a misconception of how it should work.

Kelketek committed:
Use studio_view argument with Xblock-utils.

Also make one of the tests that used Python literals use JSON while XBlock is in flux.

mtyaka commented:
@Kelketek Thanks for the explanation, makes sense. This PR looks good to me.
